lib/rules/one-var.js

```diff
@@ -250,6 +254,88 @@ module.exports = {
         }
 
 
+        /**
+         * Does the node have a `;` after it?
+         * @param   {ASTNode}  node The node to check
+         * @returns {boolean}       If the node has a `;` after it
+         */
+        function hasSemi(node) {
+            return sourceCode.getLastToken(node).type === "semi";
+        }
+
+        /**
+         * Fixer to split a VariableDeclaration into individual declarations
+         * @param   {VariableDeclaration}   declaration The `VariableDeclaration` to split
+         * @param   {?Function}             filter      Function to filter the declarations
+         * @returns {Function}                          The fixer function
+         */
+        function splitDeclarations(declaration, filter) {
+            if (!filter) {
+                filter = () => true;
+            }
+            return function *fix(fixer) {
+                for (const declarator of declaration.declarations) {
+                    const comma = sourceCode.getTokenAfter(declarator);
+
+                    if (filter(declarator) && comma.value === ",") {
+                        let spaceAfter = "";
+
+                        if (sourceCode.getTokenAfter(comma).range[0] === comma.range[1]) {
+
+                            /*
+                             * `var x,y`
+                             * comma ^^ token after comma
+                             */
+                            spaceAfter = " ";
+                        }
+                        yield fixer.replaceText(comma, `; ${declaration.kind}${spaceAfter}`);
+                    }
+                }
+            };
+        }
+
+        /**
+         * Fixer to merge a VariableDeclaration into individual declarations
+         * @param   {...VariableDeclaration} declarations The `VariableDeclarations` to merge
+         * @returns {Function}                            The fixer function
+         */
+        function mergeDeclarations() { // eslint-disable-line no-unused-vars
```

Comment:
I think it would be better to remove the unused code for now, and then add it afterwards once it's complete.